src/app/components/AlertBox/index.tsx

```diff
@@ -61,12 +61,16 @@ export function AlertBox(props: Props) {
       background={{ color: mapStatus[props.status].background }}
       pad={{ horizontal: 'small' }}
     >
-      <Box direction="row" gap="small" align="center" justify={props.center ? 'center' : 'start'}>
-        {props.icon && <Text color={color}>{props.icon}</Text>}
-        <Text weight="bold" size="12px" style={{ marginTop: 10, marginBottom: 10 }}>
-          {props.children}
-        </Text>
-      </Box>
+      {!props.children && (props.icon || props.content) && (
+        <Box direction="row" gap="small" align="center" justify={props.center ? 'center' : 'start'}>
+          {props.icon && <Text color={color}>{props.icon}</Text>}
+          <Text weight="bold" size="12px" style={{ marginTop: 10, marginBottom: 10 }}>
+            {props.content}
+          </Text>
+        </Box>
+      )}
+      {/* eslint-disable-next-line no-restricted-syntax */}
```

Comment:
Why are you disabling lint rule here? you can pass children as a text node <>Foo</> and Google Translate won't work, right?

Reply:
These are custom rules, it does not work nicely with "dynamic" content. I marked children as `ReactElement | Iterable<React.ReactNode>`, so no direct `string` is allowed. As for why, Google translate changes the structure of the elements, and that is the reason why text should be wrapped inside some element, to not break layout. As far as I know Google Translate will still work.

Reply:
why we can't just write  `{props.children}`. Why we need ` {props.children && <>{props.children}</>}` ?

Reply:
Yeah, good point. Now that props are "safeguarded" by TypeScript, should be fine that way.